### PR TITLE
Update BLT to latest `master`

### PR DESCRIPTION
**Pull request by davidbeckingsale**
- [x] Update CUDA flags
- [x] Update host configs

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
@rhornung67 - I need to update the host-configs. I think a lot of what's in there for CUDA is now redundant, and we can actually use BLT/CMake to add things like the `std` flag and OpenMP automatically.

**Comment by davidbeckingsale** [comment]
@rhornung67 - give the new blueos/nvcc_gcc host config a shot.

**Comment by rhornung67** [comment]
@davidbeckingsale I'll try out your changes later tonight or tomorrow morning. I added the BLT update to the agenda for tomorrow's meeting. After we decide what we want to change things, I can plow through the changes.

File: scripts/lc-builds/blueos_xl-2018.11.26_omptarget.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ cmake \
   -C ../host-configs/lc-builds/blueos/xl_2018_11_26.cmake \
   -DENABLE_OPENMP=On \
   -DENABLE_TARGET_OPENMP=On \
-  -DOpenMP_CXX_FLAGS="-qoffload;-qsmp=omp;-qnoeh;-qnoinline;-qalias=noansi" \
+  -DOpenMP_CXX_FLAGS="-std=c++11;-qoffload;-qsmp=omp;-qnoeh;-qnoinline;-qalias=noansi" \
```
Comment (davidbeckingsale) [inline scripts/lc-builds/blueos_xl-2018.11.26_omptarget.sh]: Is `-std=c++11` really required as an OpenMP flag?

**Comment by davidbeckingsale** [comment]
Okay, we can definitely merge this now and then do the BLT changes  later.

File: scripts/lc-builds/blueos_xl-2018.11.26_omptarget.sh (hunk 1 above)
Comment (rhornung67) [reply scripts/lc-builds/blueos_xl-2018.11.26_omptarget.sh]: Good catch.  I'lll fix that when I get back to my office after another meeting.

**Comment by rhornung67** [comment]
@davidbeckingsale I just pushed my last change. Should I merge when all tests pass?

**Comment by davidbeckingsale** [comment]
Sure thing - go ahead and review, then merge. I can't approve the PR since I opened it! 😄

**Comment by rhornung67** [comment]
@davidbeckingsale I have a few more mods to make it easier to test features for Sierra vs. EA hardware. When it passes tests, I will merge.

**Pull request merged**